### DISCUSSION TITLE
Welcome to dao Discussions!

### DISCUSSION THREAD

**Discussion by daokoder** [daokoder/dao#549] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by dumblob** [comment]
Great to see some activity again! I hope you're doing well @daokoder with Craftica etc.

It's been a long time and the world of languages "went nuts" in the meantime. If there is any interest, we can discuss some ideas which might potentially influence future Dao (if any).

**Reply by daokoder** [reply]
Wow, you are so quick ;). I am just writing a new topic about what I am considering to do with Dao for discussion. I haven't finished it yet, interrupted by responding to a comment on reddit.

The world of languages indeed "went nuts". I am glad you are back, and very interested to hear your ideas!
